### PR TITLE
feat: open changelog page when user downloaded a new version

### PR DESCRIPTION
**Pull request by erikas-taroza**
Updated UI and added function to open to change log if the user downloaded a new version.

### CODE REVIEW

**Comment by erikas-taroza** [comment]
Please test if you are able to open up to the change log window after the version has been changed.
You can test by editing the GetVersion() method to return a different string.

File: Assets/Mirror/Editor/WelcomeWindow/WelcomeWindow.cs [hunk 1]
```diff
@@ -33,40 +31,54 @@ public class WelcomeWindow : EditorWindow
         #endregion
 
         //window size of the welcome screen
-        private static Vector2 windowSize = new Vector2(500, 600);
+        private static Vector2 windowSize = new Vector2(500, 415);
 
-        //get the start up key
+        //editorprefs keys
         private static string firstStartUpKey = string.Empty;
+        private static string firstTimeMirrorKey = "MirrorWelcome";
```
Comment (Lymdun) [inline Assets/Mirror/Editor/WelcomeWindow/WelcomeWindow.cs:38]: should be readonly

File: Assets/Mirror/Editor/WelcomeWindow/WelcomeWindow.cs [hunk 2]
```diff
@@ -12,17 +12,15 @@
...
     public class WelcomeWindow : EditorWindow
     {
         #region Setup
 
         #region Urls
 
-        private const string WelcomePageUrl = "https://mirrorng.github.io/MirrorNG/";
+        private const string WelcomePageUrl = "https://mirrorng.github.io/MirrorNG/index.html";
         private const string QuickStartUrl = "https://mirrorng.github.io/MirrorNG/Articles/Guides/CommunityGuides/MirrorQuickStartGuide/index.html";
         private const string ChangelogUrl = "https://github.com/MirrorNG/MirrorNG/commits/master";
```
Comment (Lymdun) [inline Assets/Mirror/Editor/WelcomeWindow/WelcomeWindow.cs:25]: would be nice to use https://github.com/MirrorNG/MirrorNG/blob/master/Assets/Mirror/CHANGELOG.md instead

**Review by Lymdun — CHANGES_REQUESTED** [review]

File: Assets/Mirror/Editor/WelcomeWindow/WelcomeWindow.cs (hunk 2 above)
Comment (erikas-taroza) [reply Assets/Mirror/Editor/WelcomeWindow/WelcomeWindow.cs:25]: The UIToolkit doesn't support rich text. Do you want me to parse it so it is just plain text?

**Comment by erikas-taroza** [comment]
@Lymdun Please test if the welcome window does the following:
1. Open to changelog if there is a new version.
2. After closing, reopening should bring you to the welcome page and not show ``Change Log (updated)`` in the change log page.
3. First time opening (change ``firstTimeMirrorKey`` key) brings you to the welcome page.

**Review by Lymdun — APPROVED** [review]

**Pull request merged**